providers/cncf/kubernetes/src/airflow/providers/cncf/kubernetes/executors/local_kubernetes_executor.py

```diff
@@ -119,6 +120,9 @@ def job_id(self, value: int | str | None) -> None:
 
     def start(self) -> None:
         """Start local and kubernetes executor."""
+        if AIRFLOW_V_3_0_PLUS:
+            raise RuntimeError(f"{self.__class__.__name__} does not support Airflow 3.0+.")
```

Comment:
I think we should add "Use Hybrid executors instead http://linktodochere".

Reply:
Good point! Adjusted.